src/service.rs

```diff
@@ -88,7 +88,7 @@ pub struct AccountDataStored<'a> {
 pub struct Account {
     main_key: [u8; SYMMETRIC_KEY_SIZE],
     version: u8,
-    pub user: LoginResponseUser,
+    user: LoginResponseUser,
```

Comment:
Technically a breaking change, but I haven't been able to find anything that actually uses this field.

Reply:
I'm fine with this one.